java/src/jmri/implementation/AbstractTurnout.java

```diff
@@ -134,6 +139,61 @@ public void setCommandedState(int s) {
      */
     public static int DELAYED_FEEDBACK_INTERVAL = 4000;
 
+    /**
+     * Duration in Milliseconds of interval between separate Turnout commands.
+     * Note: Not updated after initialisation, which is not an issue as only used in logging. Timing is done via the TurnoutManager.
+     * <p>
+     * Change from e.g. LnTurnout extensions and scripts using {@link #setOutputInterval(int)}
+     */
+    private int TURNOUT_INTERVAL = InstanceManager.turnoutManagerInstance().getOutputInterval(mSystemName);
+
+    /**
+     * Set the delay as configured for the connection of this Turnout.
+     *
+     * @param newInterval the delay in Milliseconds
+     */
+    public void setOutputInterval(int newInterval) {
+        TURNOUT_INTERVAL = newInterval;
+        log.debug("(jmri.implementation.AbstractTurnout_TURNOUT_INTERVAL set to: {}", newInterval);
+    }
+
+    protected Thread thr;
+    protected Runnable r;
+    protected LocalTime nextWait;
+
+    /** {@inheritDoc}
+     * Used in {@link jmri.implementation.DefaultRoute#setRoute()} and
+     * {@link jmri.implementation.MatrixSignalMast#updateOutputs(char[])}.
+     */
+    @Override
+    public void setCommandedStateAtInterval(int s) {
+        nextWait = InstanceManager.turnoutManagerInstance().outputIntervalEnds(mSystemName);
```

Comment:
This assumes the delay on the default TurnoutManager is the only delay is to be used. Turnouts need to use only the manager that matches the turnout to get the delay, not whichever is the default.

Reply:
I copied this from ``private final String closedText = InstanceManager.turnoutManagerInstance().getClosedText();`` in line 58 which seems to get the appropriate Closed description from the matching TurnoutManger, not the default.
Is that not the case?